### ISSUE TITLE
value_type day for day equality match on age

### ISSUE THREAD

**Issue by kapilt** [capitalone/cloud-custodian#384]

**Comment by kapilt** [comment]
doesn't quite work, since age is a value filter, but we could zero out minute, hour, second, microsecond. need to verify no use cases for those, but seems reasonable at first glance.

**Comment by skurfer** [comment]
The goal is to be able to filter for things that are “X days old”, so maybe take exactly X days from now and see if the creation was +/-12 hours from that, or base it on midnight as suggested above.

**Comment by kapilt** [comment]
yeah. basically right now if we do an age comparison, we'll match on the current timestamp compared to the resource age, but folks want to be able to send email at 45 days, and 60 days, and then nuke at 90, and they have to do some gymnastics with greater than / less than multiple blocks to get that right.

**Comment by kapilt** [comment]
where was if we could strip the hours/minutes/seconds they could do an equality match on the integer number of days between the two timestamps.
